### ISSUE TITLE
title is showing as  audible-escape

### ISSUE THREAD

**Issue by klyver1** [macr0dev/Audiobooks.bundle#40]
it is hard to se if it right book when you scan  when it gives you title is showing as  audible escape as a title

**Comment by macr0dev** [comment]
I'm sorry, I'm not following what you're saying.

**Comment by brandonscript** [comment]
Certain books appear to be showing as "AUDIBLE ESCAPES" in the title, even though it's a correct match. When I find another example, I'll add details here.

**Comment by brandonscript** [comment]
OK here's an example.

Filename: `~/Quinn, Kate/The Empress of Rome Series/04 - Lady of the Eternal City (2015)/LadyoftheEternalCityEmpressofRomeBook4_ep6.mp3`


id3 tag info:

```
title: Lady of the Eternal City: Empress of Rome, Book 4
artist: Kate Quinn
album: Lady of the Eternal City (Unabridged)
```

Album and Sort Album names both appear as `        AUDIBLE ESCAPE    `

Match: https://www.audible.com/pd/Lady-of-the-Eternal-City-Audiobook/B00W2URG1C?qid=1564627802&sr=1-1&pf_rd_p=e81b7c27-6880-467a-b5a7-13cef5d729fe&pf_rd_r=CEH2QTG05VGVGBRPZNNH&ref=a_search_c3_lProduct_1_1 
(you can see if you search for the title, it appears with an "AUDIBLE ESCAPE" tag next to the search result)

**Comment by macr0dev** [comment]
Ah.  I see what you're talking about.  I don't think I've every matched a book that's part of one of their promotional packages before.  They just slipped it in above the title and it superseded it in the search.  Looks like you've found the work around.  I've make a change on my test server and it appears to be working just fine.  I've updated the code to be more specific in grabbing the title to make sure it skips over the "AUDIBLE ESCAPE" tag on the page.

Thanks for bringing this up.

**Comment by brandonscript** [comment]
I sent in a PR that fixes #40 and #42 if you want 🤷‍♂️

**Comment by macr0dev** [comment]
I used a slightly different tag than you for #40, and I'm fixing to merge your #42 in.
